base.py

```diff
@@ -377,6 +400,29 @@ def post_create_table(self, table):
                 ", ".join(self.denormalize_column_name(key) for key in cluster))
         return text
 
+    def visit_create_stage(self, create_stage, **kw):
+        """
+        This visitor will create the SQL representation for a CREATE STAGE command.
+        """
+        return "CREATE OR REPLACE STAGE {}{} URL={}".format(
+            create_stage.stage.namespace,
+            create_stage.stage.name,
+            create_stage.container.__repr__())
+
+    def visit_create_file_format(self, file_format, **kw):
+        """
+        This visitor will create the SQL representation for a CREATE FILE FORMAT
+        command.
+        """
+        return "CREATE OR REPLACE FILE FORMAT {} TYPE='{}' ".format(
+            file_format.format_name,
+            file_format.formatter.file_format
+        ) + " " .join(
```

Comment:
Agreed, added the option array as last parameter of the format string.
I didn't use f-strings on purpose since the existing code used `format` or the `%` operator as well, so I'm not sure if you had any design considerations on backwards compatibility for Python < 3.6.